### DISCUSSION TITLE
Phone numbers

### DISCUSSION THREAD

**Discussion by snan** [jekyll/jekyll#9198] [Q&A]
How can I prevent kramdown from turning number sequences into `<a href="tel:` hyper links? I just wanna write about dates and integer sequences in a non-phone context.

**Comment by Alexander01998** [comment]
Writing it like this should work: `<span>5550123456</span>`

Though I can't seem to reproduce the issue in my Jekyll installation. Even if I just write `5550123456` or `+5550123456`, kramdown seems to render it as normal text.

**Comment by snan** [comment]
I've been filing bugs all over the place and also trying to fix the issue with Jekyll and Kramdown and Pandoc that they mark up numbers (like integer sequences and math stuff) as phone numbers.

Turns out they don't do that.

It's the iPad version of Safari that does. 🤦🏻‍♀️

It can't even call any numbers so what the heck

For example 555-0123456
